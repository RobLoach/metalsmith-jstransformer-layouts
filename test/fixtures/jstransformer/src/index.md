---
layout: layouts/_layout.jade
title: Hello World
---

# Hello World

This is index.md.
