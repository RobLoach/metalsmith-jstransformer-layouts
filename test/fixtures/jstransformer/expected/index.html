<!DOCTYPE html><html lang="en"><head><title>Hello World</title></head><body><h1 id="hello-world">Hello World</h1>
<p>This is index.md.</p>
</body></html>